AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -121,6 +121,7 @@ private var runtimeTargets: [Target] {
             dependencies: [
```

Comment:
As you can see below, many unused dependencies were eliminated in the runtime modules.  One was an implicit dependency that was added.